packages/generator/src/generator.ts

```diff
@@ -111,12 +111,45 @@ function replaceJsxConditionals(program: Collection<j.Program>, templateValues:
   return program
 }
 
+export type Prompt = {
```

Comment:
suggestion: Pass a generic in this type to set the type of choices
```
export interface Prompt<T> {
  name: string
  message: string
  choices: T[];
}

Reply:
also nit: Prefer the use of `interface` over `type` because interfaces [can be extended](https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases) whereas types cannot